test/runner/listeners.js

```diff
@@ -53,6 +53,14 @@
 					return nu;
 				}
 			}
+
+			// Serialize Symbols as string representations so they are
+			// sent over the wire after being stringified.
+			if ( typeof value === "symbol" ) {
+				var ctor = Symbol.keyFor( value ) !== undefined ? "Symbol.for" : "Symbol";
+				return ctor + "(" + JSON.stringify( value.description ) + ")";
+			}
```

Comment:
Note that `JSON.stringify` also fails to round-trip bigints, NaN, ±Infinity, and undefined. If we start caring about those cases, we should switch to a higher-order encoding similar to [Endo "Smallcaps"](https://github.com/endojs/endo/blob/1813f35432e266a4998c090f32b184b166d5d47f/packages/marshal/src/encodeToSmallcaps.js#L47-L67) (_in which e.g. the array `[0, 42n, NaN, -Infinity, undefined, "#"]` would be encoded as `[0, "+42", "#NaN", "#-Infinity", "#undefined", "!#"]`_).

Reply:
I don't think we need to go that far.